app/views/evidence_hub/_thematic_review_message.html.erb

```diff
@@ -0,0 +1,8 @@
+<p class='evidence-hub__thematic-review-message'>
+<% link = link_to(
+            thematic_review.title,
+            thematic_review_path(locale: params[:locale], id: thematic_review.slug)
+          )
+ %>
+  <%= raw I18n.t('fincap.evidence_hub.thematic_review.message_box_info', link: link) %>
```

Comment:
We've addressed html safe text in translation files in different ways across the mas repos. Is there a preference/reason to doing it in the view as above or in the translation file as per the approach in wpcc for example, https://github.com/moneyadviceservice/wpcc/blob/master/config/locales/en.yml#L44? @johnplayer1982 @davidtrussler @rhianwatkins1 @willhall88

Reply:
@margOnline Sorry, I have no idea on best practice here but I guess we should be consistent.

Reply:
I don't have a particularly strong opinion either way. I think perhaps this solution is a more appeasing one?